src/main/java/org/pesmypetcare/webservice/entity/EventEntity.java

```diff
@@ -0,0 +1,87 @@
+package org.pesmypetcare.webservice.entity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.EventReminder;
+import lombok.Data;
+import com.google.api.services.calendar.model.Event;
+import org.pesmypetcare.webservice.jsonhandler.DateTimeHandler;
+
+import java.util.Arrays;
+
+/**
+ * @author Marc Simó
+ */
+@Data
+public class EventEntity {
+    private String id;
+    private String summary;
+    private String location;
+    private String description;
+    private String color;
+    private Integer emailReminderMinutes;
+    private Integer repetitionInterval;
+    @JsonDeserialize(using = DateTimeHandler.class)
+    private DateTime startDate;
+    @JsonDeserialize(using = DateTimeHandler.class)
+    private DateTime endDate;
+
+    public EventEntity() {}
+
+    public EventEntity(Event event) {
+        id = event.getId();
+        summary = event.getSummary();
+        location = event.getLocation();
+        description = event.getDescription();
+        color = event.getColorId();
+        startDate = event.getStart().getDateTime();
+        endDate = event.getEnd().getDateTime();
+    }
+
+    public EventEntity(String id, String summary, String description, String location, String color,
+                         Integer emailReminderMinutes,
+                       Integer repetitionInterval, DateTime startDate, DateTime endDate) {
+        this.id = id;
+        this.summary = summary;
+        this.location = location;
+        this.description = description;
+        this.color = color;
+        this.emailReminderMinutes = emailReminderMinutes;
+        this.repetitionInterval = repetitionInterval;
+        this.startDate = startDate;
+        this.endDate = endDate;
+    }
+
+    public Event convertToEvent() {
+        Event event = new Event()
+            .setId(id)
+            .setSummary(summary)
+            .setLocation(location)
+            .setDescription(description)
+            .setColorId(color);
+
+        EventDateTime start = new EventDateTime()
+            .setDateTime(startDate)
+            .setTimeZone("Europe/Madrid");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Europe/Madrid" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)